apps/seven/registry.config.ts

```diff
@@ -4,6 +4,7 @@ const addons = [
   '@plone/layout',
   '@plone/cmsui',
   '@plone/publicui',
+  '@plone/agave',
```

Comment:
Not 100% sure that Agave has to be default... We can consider this in the future though.